lib/eco/treelexer/lexer.py

```diff
@@ -181,18 +181,22 @@ def append(self):
             self.result.append(self.char())
 
     def get_token(self):
```

Comment:
Can we document this method? Its name doesn't seem to quite match its functionality?